payloadhandler.py

```diff
@@ -0,0 +1,199 @@
+import random
+import re
+from copy import copy
+from errorlogparser import ServoErrorLogParser
+
+
+class PayloadHandler():
+    def handle_payload(self, payload):
+        raise NotImplementedError
+
+
+class TravisPayloadHandler(PayloadHandler):
+    msg_template = "Please fix the following error:\n\n**File:** {}\n**Line Number:** {}\n**Error:** {}"
```

Comment:
Note to self: we should improve the friendliness of this message before going live with this.